src/components/DailyStatistics/TransactionHistoryCard.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FunctionComponent } from "react";
+import { size, fontSize } from "../../common/styles";
+import { StyleSheet, View } from "react-native";
+import { AppText } from "../Layout/AppText";
+import { Card } from "../Layout/Card";
+
+interface TransactionHistoryCardComponent {
+  transactionHistory: { name: string; category: string; quantity: number }[];
+}
+
+const styles = StyleSheet.create({
+  stats: {
+    flexDirection: "column",
+    marginTop: -size(0.5),
+    minHeight: "20%"
+  },
+  categoryName: {
+    fontFamily: "brand-bold",
+    fontSize: fontSize(0),
+    marginBottom: size(3)
+  },
+  quantity: {
+    fontFamily: "brand-bold",
+    fontSize: fontSize(0),
+    marginBottom: size(3)
+  },
+  transactionText: {
+    flexDirection: "row",
+    display: "flex",
+    justifyContent: "space-between"
+  },
+  noTransactionText: {
+    fontFamily: "brand-bold",
+    fontSize: fontSize(0),
+    marginBottom: size(3),
+    flexDirection: "row",
+    textAlign: "center"
+  }
+});
+
+export const TransactionHistoryCardComponent: FunctionComponent<TransactionHistoryCardComponent> = ({
+  transactionHistory
+}) => {
+  return (
+    <Card style={styles.stats}>
+      {transactionHistory.length !== 0 ? (
+        transactionHistory.map(item => (
+          <View style={styles.transactionText} key={item.category}>
+            <AppText style={styles.categoryName}>{item.name}</AppText>
+            <View style={styles.transactionText}>
+              <AppText style={styles.quantity}>
+                {item.quantity.toLocaleString()} qty
```

Comment:
Eh so what about those which quantities we have labels for? Like $5? Saw in story we are supposed to cater for that

Also is this actually the same display as past transactions? If yes, we can reuse the components?

Reply:
thanks! i've added the units as well, and also confirmed with Eida that each dollar will be considered 1 count as well. 
i.e. $300 vouchers and 50 potatoes, total count = 350 distributed

in addition, the past transactions display and stats display for now is different, because we are thinking of adding location into the stats display as well, so its still up for discussion on how we wanna approach this, so im thinking for now we separate them two first?

Reply:
Location also supposed to be in past transactions display, just that no such feature now, so it's not in yet. So yeah they are the same haha. Can confirm with Eida.

Reply:
okay will confirm with her on this! for now will keep it as is?

Reply:
Only concern is whether our current process enables to clean up all these broken windows :)